build-logic/jvm/src/main/kotlin/build-logic.java.gradle.kts

```diff
@@ -0,0 +1,45 @@
+import com.github.vlsi.gradle.dsl.configureEach
+
+plugins {
+    id("java")
+    id("ch.tutteli.gradle.plugins.junitjacoco")
+    id("com.github.vlsi.crlf")
+    id("com.github.vlsi.gradle-extensions")
```

Comment:
@vlsi I like the improved error reporting in case a gradle build fails due to some misconfiguration or whatever. Yet, it is too verbose regarding tests for my taste. Is there a way to disable the report for the individual tests? This basically clutters the output too much, having ~4500 tests for atrium-api-fluent alone. The summary would be enough (in this case I could disable the one of tutteli-plugin as yours looks nicer :slightly_smiling_face:)
![image](https://github.com/robstoll/atrium/assets/5557885/c92f3106-4980-4dca-afe3-faa0015b7f65)

Reply:
How about the following?

```
# com.github.vlsi.gradle-extensions prints only failing or slow test results
slowTestLogThreshold=1000
slowSuiteLogThreshold=2000
```

Reply:
I don't think the problem are slow tests, most prints mention 0.0sec
maybe the problem is that it prints a summary per suite as well?

Reply:
The properties above are the thresholds, so if you increase them, it won't print tests that fit into the thresholds. The defaults are 0, 0 which might not be a best selection for projects that have many tests.

I think I would increase the default thresholds from 0, 0 to something less verbose.
The key idea there was to catch cases when a single test takes significant time.

Here's sample output in jqwik:
* Output: https://github.com/jqwik-team/jqwik/actions/runs/5302500735/jobs/9597464437#step:5:407
* Configuration: https://github.com/jqwik-team/jqwik/blob/e2e3e0c450a46d87cdd59047c5d97ced15e73b4b/gradle.properties#L1-L3

Reply:
Ah ok, I was so sure that the default threshold would be above 0 that I did not even try it out. Sorry about that, going to adjust it then. I see the idea behind the threshold and IMO it makes sense to have something like that